server/planning/assignments/assignments_unlink_test.py

```diff
@@ -5,9 +5,32 @@
 
 
 class AssignmentUnlinkTestCase(TestCase):
+    def setUp(self):
+        super().setUp()
+        with self.app.app_context():
+            users = [
+                {
+                    "_id": ObjectId("5d385f31fe985ec67a0ca583"),
```

Comment:
would be good to store that as a constant, it's kind of hard to check later all those ids are matching